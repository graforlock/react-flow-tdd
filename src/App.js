// @flow
import React, { Component } from 'react'
import './App.css'

import IdeaSection from './components/IdeaSection'

type Props = {}

class App extends Component<Props> {
  render () {
    return (
      <main className='App'>
        <header className='App-header' />
        <IdeaSection />
      </main>
    )
  }
}

export default App
