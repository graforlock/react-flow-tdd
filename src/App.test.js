// @flow
import React from 'react'
import { shallow } from 'enzyme'

import App from './App'

describe('App', () => {
  let wrapper

  beforeEach(() => {
    wrapper = <App />
  })

  it('renders component and matches current snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
