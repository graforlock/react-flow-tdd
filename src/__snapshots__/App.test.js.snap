// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders component and matches current snapshot 1`] = `<App />`;
